modules/ipi-install-bmc-addressing-for-dell-idrac.adoc

```diff
@@ -80,8 +80,6 @@ platform:
 
 [NOTE]
 ====
-There is a known issue on Dell iDRAC 9 with firmware version `04.40.00.00` or later for installer-provisioned installations on bare metal deployments. The Virtual Console plugin defaults to eHTML5, an enhanced version of HTML5, which causes problems with the *InsertVirtualMedia* workflow. Set the plugin to use HTML5 to avoid this issue. The menu path is *Configuration* -> *Virtual console* -> *Plug-in Type* -> *HTML5* .
-
 Ensure the {product-title} cluster nodes have *AutoAttach* enabled through the iDRAC console. The menu path is: *Configuration* -> *Virtual Media* -> *Attach Mode* -> *AutoAttach* .
```

Comment:
Q to SME: Should I separate, what seems to me, to be two notes? At the moment, they exist in one note.

Reply:
yes please! :)

Reply:
Thanks! Updated.